android/java/org/chromium/chrome/browser/rewards/BraveRewardsTipConfirmationFragment.java

```diff
@@ -0,0 +1,186 @@
+    private BraveRewardsNativeWorker mBraveRewardsNativeWorker;
+    private int mStatus;
+    private String mPublisherName;
+    private boolean mMonthlyContribution;
+    private TextView mNextTipDateText;
+    private TextView mNextTipDateValue;
+
+    public static BraveRewardsTipConfirmationFragment newInstance(
+            int status, String name, double amount, boolean isMonthly) {
+        BraveRewardsTipConfirmationFragment fragment = new BraveRewardsTipConfirmationFragment();
+        Bundle args = new Bundle();
+        args.putDouble(BraveRewardsSiteBannerActivity.AMOUNT_EXTRA, amount);
+        args.putBoolean(BraveRewardsSiteBannerActivity.IS_MONTHLY_CONTRIBUTION, isMonthly);
+        args.putInt(BraveRewardsSiteBannerActivity.STATUS_ARGS, status);
+        args.putString(BraveRewardsSiteBannerActivity.NAME_ARGS, name);
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Override
+    public View onCreateView(
+            LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        mBraveRewardsNativeWorker = BraveRewardsNativeWorker.getInstance();
+        mBraveRewardsNativeWorker.AddObserver(this);
+        View view = inflater.inflate(R.layout.brave_rewards_tip_confirmation, container, false);
+        if (getArguments() != null) {
+            double amount = getArguments().getDouble(BraveRewardsSiteBannerActivity.AMOUNT_EXTRA);
+            mMonthlyContribution = getArguments().getBoolean(
+                    BraveRewardsSiteBannerActivity.IS_MONTHLY_CONTRIBUTION);
+            int status = getArguments().getInt(BraveRewardsSiteBannerActivity.STATUS_ARGS, -1);
+            String publisherName =
+                    getArguments().getString(BraveRewardsSiteBannerActivity.NAME_ARGS);
+
+            initView(view, status, publisherName, amount, mMonthlyContribution);
+        }
+        return view;
+    }
+
+    @SuppressLint("SetTextI18n")
+    private void initView(
+            View view, int status, String name, double amount, boolean monthlyContribution) {
+        mStatus = status;
+        mPublisherName = name;
+        TextView tipAmountTextView = view.findViewById(R.id.amount_sent);
+        tipAmountTextView.setText(amount + " BAT");
+        TextView tipTypeDescriptionTextView = view.findViewById(R.id.your_tip_amount_sent);
+        TextView oneTtimeTipNote = view.findViewById(R.id.one_time_tip_note);
+        String oneTimeNoteString = getResources().getString(R.string.one_time_tip_success_note);
+
+        final StringBuilder sb1 = new StringBuilder();
+        sb1.append("<b>");
+        sb1.append(getResources().getString(R.string.note_text));
+        sb1.append("</b>");
+        sb1.append(" ");
+        sb1.append(oneTimeNoteString);
+        Spanned toInsert = BraveRewardsHelper.spannedFromHtmlString(sb1.toString());
+        oneTtimeTipNote.setText(toInsert);
```

Comment:
can use `android:textStyle="bold` instead of html formatting

Reply:
in this same textview having different textStyle that's why used this way.